### PR TITLE
Restoring/removing integration tests

### PR DESCRIPTION
**Pull request by ivanbasov**
1. Removing integration tests that both
(a) are commented due to open issues
(b) have no infrastructure support in VisualStudioIntegrationTestsUtilities
those tests will be kept in Tao tests for a while (as suggested by @jasonmalinowski)

2. Removing CodeActions tests that involve usings already referenced in interactive, i.e. those scenarios are obsolete. Will remove them also from Tao tests.

3. Restoring tests that were disabled due to insufficient infrastructure in test framework. Those scenarios should be fine for now.

@dotnet/roslyn-ide

### CODE REVIEW

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs [hunk 1]
```diff
@@ -24,69 +24,20 @@ public new void Dispose()
             base.Dispose();
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/801")]
```
Comment (jinujoseph) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs]: I see 801 still open , is reenabling this test intentional ?

Comment (ivanbasov) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs]: yes, it seem to be unrelated with #801
